### DISCUSSION TITLE
My Norton antivirus thinks Liberation 5.1 .EXE is a Trojan ???

### DISCUSSION THREAD

**Discussion by JF-JEFX** [Khopa/dcs_liberation#1963] [Q&A] [answered]
Hello, I dont know if anyone else has had this problem, but this morning, my Norton antivirus (360) has automatically erased my Liberation 5.1 EXE saying it is a Trojan.Gen.9!  I tried redownloading and reinstalling and it did the same thing, adding that it might be also a SONAR.Dropper!Gen2 virus...  I tried contacting Norton but their customer service is really slow and bad... on top of that, if... Norton detects something like that, it might be that there is something with the .EXE that is not quite right?  anyone knows about this or what shall I do?  (by the way, we are not even talking about executing Liberation in Admin mode, which I use to do all the time... not only I cannot even start it, it gets erased before I can even try...).
thanks
JEFX

**Comment by Starfire13** [comment, marked as answer]
Liberation is open source you are free to review the code. I would suggest not using Norton Antivirus and use a better antivirus, like Windows Defender.
